zigbee: Refactor logging ZBOSS traces (#5279)
nrfconnect/sdk-nrf: samples/zigbee/ncp/README.rst

.. note::
    The USB is used as the default NCP communication channel when using the nRF52840 Dongle.

> You can configure :ref: Zigbee stack logs`<zigbee_ug_logging_stack_logs>` to be printed in binary form over USB using independent serial port.

Reviewer: ```suggestion
You can configure :ref:`Zigbee stack logs <zigbee_ug_logging_stack_logs>` to be printed in binary format over the USB CDC using an independent serial port.
```

> Once configured, no additional code is required.

Reviewer: Code required where? What does this mean specifically?

Author: For NCP sample, this can be configured purely by setting correct Kconfig options - no changes in code are required.
